consts.php

```diff
@@ -106,3 +106,13 @@
         'silverstripe-installer' => '5',
     ],
 ];
+
+const INSTALLER_TO_REPO_MINOR_VERSIONS = [
+    '4.10' => [
```

Comment:
What's the reasoning behind this structure? Seems like `[repo => [repo-version => installer-version]]` makes more intuitive sense?
You can then just go
```php
if (isset(INSTALLER_TO_REPO_MINOR_VERSIONS[$repo][$branch]) {
    return INSTALLER_TO_REPO_MINOR_VERSIONS[$repo][$branch] . '.x-dev';
}
```

Reply:
Done this way because it's more intuitive for a human to update the const

The more convoluted code is a worthwhile trade

Reply:
Fair enough, I'm happy with that.